FC_Board/lib/functions.py

```diff
@@ -15,25 +15,32 @@
 from packet_manager import PacketManager
 from packet_sender import PacketSender
 
+try:
+    from typing import List, Dict, OrderedDict, Literal, Union, Any
+    import circuitpython_typing
+except:
+    pass
+from pysquared import Satellite
+
 
 class functions:
 
-    def debug_print(self, statement):
+    def debug_print(self, statement: str) -> None:
         if self.debug:
             print(co("[Functions]" + str(statement), "green", "bold"))
```

Comment:
It looks like `statement` has a type conversion to `str` on line 30. It almost looks like a bug but since type hints aren't enforced at runtime this feels fair for now.

Reply:
I got held up on this function a while because of the conversion there on line 30. I saw that every time the function is called, there is a string passed. However, line 30 made me question if I should change statement to just be of type 'Any'. This was a tough one